src/catalog/catalog_entry/view_catalog_entry.cpp

```diff
@@ -64,4 +64,19 @@ string ViewCatalogEntry::ToSQL() {
 	return sql + "\n;";
 }
 
+unique_ptr<CatalogEntry> ViewCatalogEntry::Copy(ClientContext &context) {
+	auto create_info = make_unique<CreateViewInfo>(schema->name, name);
+	create_info->query = move(query);
```

Comment:
This is not a copy, this is a move. This should be query->Copy();. Otherwise the original entry is invalidated upon calling Copy().